packages/api/src/hooks/p2p/useAdvertiserRelationsAddBlocked.ts

```diff
@@ -0,0 +1,22 @@
+import { useCallback } from 'react';
+import useInvalidateQuery from '../../useInvalidateQuery';
+import useMutation from '../../useMutation';
+
+/** This hook blocks advertisers of the current user by passing the advertiser id. */
+const useAdvertiserRelationsAddBlocked = () => {
+    const invalidate = useInvalidateQuery();
+    const { data, mutate, ...rest } = useMutation('p2p_advertiser_relations', {
+        onSuccess: () => invalidate('p2p_advertiser_relations'),
+    });
+
+    const addBlockedAdvertiser = useCallback((id: number[]) => mutate({ payload: { add_blocked: id } }), [mutate]);
+
+    return {
+        data,
+        /** Sends a request to block advertiser of the current user by passing the advertiser id. */
+        addBlockedAdvertiser,
```

Comment:
```suggestion
        mutate: addBlockedAdvertiser,
```

Reply:
Why?!

Reply:
@niloofar-deriv, this is the convention used in every other hook, but feel free to ignore this 👍

Reply:
I also feel that we should keep our conventions like this, for queries we return `data`, for mutation we return `mutation`, so developers will know that query/mutation hooks will always return the same function name

Reply:
Sure guys will change this 🙏